src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/checkout/address.yaml

```
@@ -15,11 +18,66 @@ twig_hooks:
                 template: '@SyliusShop/checkout/address/form/addresses.html.twig'
             navigation:
                 template: '@SyliusShop/checkout/address/form/navigation.html.twig'
-                
-        'sylius_shop.checkout.address.form.addresses':
+
+        'sylius_shop.checkout.address.form.addresses#billingRequired':
+            billing_address:
+                template: '@SyliusShop/checkout/address/form/addresses/address.html.twig'
+                configuration:
+                    type: 'billing'
+            different_shipping_address:
+                template: '@SyliusShop/checkout/address/form/addresses/different_address.html.twig'
+                configuration:
+                    type: 'shipping'
+                    field: 'differentShippingAddress'
+            shipping_address:
+                template: '@SyliusShop/checkout/address/form/addresses/address.html.twig'
+                configuration:
+                    type: 'shipping'
+                    isDifferentAddress: true
+        
+
+        'sylius_shop.checkout.address.form.addresses#shippingRequired':
+            shipping_address:
+                template: '@SyliusShop/checkout/address/form/addresses/address.html.twig'
+                configuration:
+                    type: 'shipping'
+            different_billing_address:
+                template: '@SyliusShop/checkout/address/form/addresses/different_address.html.twig'
+                configuration:
+                    type: 'billing'
+                    field: 'differentBillingAddress'
+            billing_address:
+                template: '@SyliusShop/checkout/address/form/addresses/address.html.twig'
+                configuration:
+                    type: 'billing'
+                    isDifferentAddress: true
+            
+        
+        'sylius_shop.checkout.address.form.addresses.address#billing':
```

Shouldn't it be
```suggestion
        'sylius_shop.checkout.address.form.addresses.billing_address':
```
to be consistent with above hooks? 🤔